src/components/InputField/component.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
-import { Form, Input } from 'antd'
+import { Form, Input as AntInput } from 'antd'
```

Comment:
why do you need alias?

Reply:
@VladWithCookies yes, because I have the default props called Input

Reply:
renamed